editor/components/rich-text/README.md

```diff
@@ -93,11 +97,8 @@ wp.blocks.registerBlockType( /* ... */, {
 	edit: function( props ) {
 		return wp.element.createElement( wp.editor.RichText, {
 			tagName: 'h2',
-			className: props.className,
```

Comment:
Maybe we can leave this sample in the docs. As we still support using value + onChange and it may be useful in some cases.

Reply:
It's hard to tell, maybe we should update only Gutenberg handbook to contain the shorter version.